pkgs/games/gnoll/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, python39, flex, bison }:
+
+stdenv.mkDerivation rec {
+  pname = "gnoll";
+  version = "4.3.4";
+
+  src = fetchFromGitHub {
+    owner = "ianfhunter";
+    repo = "GNOLL";
+    rev = "v${version}";
+    sha256 = "1s495qmakw2n50x11klf6wb2869b5v94wsihq4r19mqdbg1pxm1y";
+  };
+
+  buildInputs = [ python39 flex bison ];
+
+  installPhase = "mkdir -p $out/bin ; mv build/dice $out/bin";
```

(I think it's slightly more common to have the two shell commands on individual lines, but I have also seen your stylistic preference. This is just for your information)

---

This is wrong in all levels.
Just for start, it does not call the pre and post hooks.
```suggestion
  installPhase = ''
    runHook preInstall
    install -Dm755 build/dice $out/bin/
    runHook postInstall
  '';
```